src/common_textures.h

```diff
@@ -5,10 +5,25 @@
 #include <PR/gbi.h>
 
 typedef struct {
-    /* 0x0 */ s16 waypointId1;
-    /* 0x2 */ s16 waypointId2;
-    /* 0x4 */ s32 unk4;
-} UnkCommonTextureStruct0; // size = 0x8
+    /* 0x0 */ s16 waypointStart;
+    /* 0x2 */ s16 waypointEnd;
+    /* 0x4 */ s32 type;
+} CPUBehaviour; // size = 0x8
+
+enum {
+    BEHAVIOUR_NONE = 0,
+    BEHAVIOUR_1,
+    BEHAVIOUR_HOP,
+    BEHAVIOUR_3,
+    BEHAVIOUR_4,
+    BEHAVIOUR_5,
+    BEHAVIOUR_NORMAL_SPEED,
+    BEHAVIOUR_FAST_SPEED,
+    BEHAVIOUR_SLOW_SPEED,
+    BEHAVIOUR_9,
+    BEHAVIOUR_10,
+    BEHAVIOUR_MAX_SPEED
+};
```

Comment:
I think it's best practice to name Enums. And then you can do something like
```c
enum CPUBehaviour {
    NONE,
    FAST,
    NORMAL,
};
```
Without having to repeat `BEHAVIOUR_` for every enum.

Reply:
the problem is if I don't put the prefix BEHAVIOUR_ we can't differenciate a FAST to another FAST because they have the same name but not necessarily the same value

Reply:
Resolving because what you did is correct.